fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -25,179 +40,113 @@ NONE_METHOD_OPT="NONE"
 # --------------------------------------------------------------------------------
 #  Usage Description Function
 # --------------------------------------------------------------------------------
+opts_SetScriptDescription "Run fMRIVolume processing"
 
-script_name=$(basename "${0}")
-
-show_usage() {
-	cat <<EOF
+opts_AddMandatory '--studyfolder' 'Path' 'path' "folder containing all subject" "--path"
 
-${script_name}: Run fMRIVolume processing pipeline
+opts_AddMandatory '--subject' 'Subject' 'subject ID' ""
 
-Usage: ${script_name} [options]
+opts_AddMandatory '--fmritcs' 'fMRITimeSeries' 'file' 'input fMRI time series (NIFTI)'
 
-  [--help] : show usage information and exit
-  --path=<path to study folder>
-  --subject=<subject ID>
-  --fmritcs=<input fMRI time series (NIFTI)>
-  --fmriname=<name (prefix) to use for the output>
-  --fmrires=<final resolution (mm) of the output data>
+opts_AddMandatory '--fmriname' 'NameOffMRI' 'string' 'name (prefix) to use for the output'
 
-  --biascorrection=<method to use for receive coil bias field correction>
+opts_AddMandatory '--fmrires' 'FinalfMRIResolution' 'number' 'final resolution (mm) of the output data'
 
-        "SEBASED"
-             use bias field derived from spin echo images, must also use --dcmethod="${SPIN_ECHO_METHOD_OPT}"
+opts_AddMandatory '--biascorrection' 'BiasCorrection' "Method for receive bias correction, accepted values are: 'SEBASED OR LEGACY OR NONE' 
```

Comment:
Delete `'SEBASED OR LEGACY OR NONE'`, it is redundant given the list below (and the quotes shouldn't contain the "or" that separates one option from another anyway).

Reply:
Done